cli/cmd/file.go

```diff
@@ -0,0 +1,30 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import "os"
+
+// fileExists checks if a file exists and is not a directory
+func fileExists(filename string) bool {
```

Comment:
Would this function not be better placed in internal directory?

Reply:
I can move it but it felt weird when I created a file package since there is a native one already that doesn't do this check out-of-the-box. 

Do you want me to move it? If so, help me with a package name.

Reply:
I understand what you mean. Let's keep it here for now, if we come to reuse this function we can tackle that then.